src/test/java/WordleTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Test;
+import runge.WordleDictionary;
+
+import java.io.*;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class WordleTest
```

Comment:
The name of a test class should match the name of the class that it's testing.
In this case it should be WordleDictionary test and should be in the same package as WordleDictionary.